.gitlab-ci.yml

```diff
@@ -0,0 +1,19 @@
+image:
+  name: "openjdk:11.0.11-9-jdk"
+
+stages:
+  - build
+  - release
+
+build:
+  stage: build
+  script:
+    - ./mvnw clean install -DskipTests
```

Comment:
github uses `./mvnw clean package -DskipTests` and `./mvnw verify` any reason why it is different here? Before https://github.com/signalfx/signalfx-java/pull/119 github used the same commands as here.

Reply:
No reason -- this one can use `package` too; I just typed those commands from the memory and not copied them 😅

Reply:
Actually github uses `./mvnw test` instead of `./mvnw verify`, meant to write it in the first comment but messed up :)

Reply:
Okay, now they're the same 😛

Reply:
I *think* `verify` is the preferred maven way to do things now, FYI.

Reply:
OK, let's use verify everywhere then 👍